apps/encryption/lib/Crypto/Crypt.php

```diff
@@ -240,10 +218,9 @@ public function generateHeader($keyFormat = self::DEFAULT_KEY_FORMAT) {
 	 * @param string $iv
 	 * @param string $passPhrase
 	 * @param string $cipher
-	 * @return string
 	 * @throws EncryptionFailedException
 	 */
-	private function encrypt($plainContent, $iv, $passPhrase = '', $cipher = self::DEFAULT_CIPHER) {
+	private function encrypt($plainContent, $iv, $passPhrase = '', $cipher = self::DEFAULT_CIPHER): string {
```

Comment:
Sorry I did not mark all of them earlier, I assumed two mentions where enough.
```suggestion
	private function encrypt(string $plainContent, $iv, string $passPhrase = '', string $cipher = self::DEFAULT_CIPHER): string {
```

That might also lead to comment removal above.